src/tests/Dictionary/DictionaryContainer.test.jsx

```diff
@@ -172,4 +173,40 @@ describe('DictionaryOverview', () => {
     </MemoryRouter>);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should handle opening of subscription modal', () => {
+    const props = {
+      dictionary: { dictionary },
+      versions: [versions, customVersion],
+      headVersion: [versions],
+      url: '',
+      dictionaryConcepts: [],
+      showEditModal: jest.fn(),
+      hideSubModal: jest.fn(),
+      showSubModal: jest.fn(),
+      handleRelease: jest.fn(),
+      match: {
+        params: {
+          ownerType: 'testing',
+          owner: 'chriskala',
+          type: 'collection',
+          name: 'chriskala',
+        },
+      },
+      fetchDictionary: jest.fn(),
+      fetchVersions: jest.fn(),
+      fetchDictionaryConcepts: jest.fn(),
+    };
+
+    const wrapper = mount(<Provider store={store}>
+      <MemoryRouter>
+        <DictionaryDetailCard {...props} />
+      </MemoryRouter></Provider>);
+
+    expect(wrapper.find('.subscription-link').at(0).exists()).toBe(true);
+    wrapper.find('.subscription-link').at(0).simulate('click');
+    expect(wrapper.find('.modal-container').at(0).exists()).toBe(true);
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Please no what exactly to expect. Too many tests in here. I think according to your `it` description, expecting the modal not to exist before click and expecting it to be there after the click is a better way to test that. What do you think? Avoid snapshots when simulating events.

Reply:
It tests whether the subscription link exists and what happens whether the modal appears when it is clicked. I, however, think the not having snapshots is valid. Will remove it.